ark/phenotyping/cell_consensus_cluster.R

```diff
@@ -0,0 +1,57 @@
+# Runs consensus clustering on the averaged cell data table, defined as the mean counts of each
+# pixel SOM/meta cluster across all cell SOM clusters in each fov (m x n table, where m is the number of
+# cell SOM/meta clusters and n is the number of pixel SOM/meta clusters)
+
+# Usage: Rscript {maxK} {cap} {cellClusterPath} {clusterAvgPath} {cellConsensusPath} {seed}
+
+# - maxK: number of consensus clusters
+# - cap: z-score scaling factor
```

Comment:
I wouldn't really call it a scaling factor, since that implies we're scaling all the z-scores by some number. Maybe something like "maximum z-score value" or something. It's also not only for visualization, we clip the z-scores before we do consensus clustering.